pwndbg/gdblib/elf.py

```diff
@@ -138,20 +152,21 @@ def get_elf_info_rebased(filepath: str, vaddr: int) -> ELFInfo:
     # silently ignores "wrong" vaddr supplied for non-PIE ELF
     load = vaddr if raw_info.is_pic else 0
     headers = dict(raw_info.header)
-    headers["e_entry"] += load
+    headers["e_entry"] += load  # type: ignore[operator]
```

Comment:
Could you explain why we need all the `# type: ignore[operator]` comments in this file? Just to make this easier for me to review.

Reply:
Thanks. Maybe this file could be refactored in the future if pyelftools ever gets type stubs (https://github.com/python/typeshed/pull/11262).